From e3405d25e66ec6970c38ca8e57dd873a25e2ff3b Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 3 May 2019 09:15:49 +0000
Subject: [PATCH] db: clear depends_* when parse APKBUILDs

Some APKBUILDs uses makedepends="$depends_dev ..." without first setting
depends_dev. This may result in depends_dev leak when parsing APKBUILDs
---
 aports/db.lua | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/aports/db.lua b/aports/db.lua
index bb7e136..2ed14bd 100644
--- a/aports/db.lua
+++ b/aports/db.lua
@@ -98,6 +98,10 @@ local function apkbuilds_open(aportsdir, repos)
 			license=
 			options=
 			depends=
+			depends_doc=
+			depends_dev=
+			depends_libs=
+			depends_openrc=
 			makedepends=
 			makedepends_build=
 			makedepends_host=
-- 
2.21.0

